README.md

```diff
@@ -1,369 +1,137 @@
+If you have changes that are not yet committed, when you switch between branches, the untracked
+changes will follow you to the next branch. These changes must be committed with `git commit -m "<my commit message>"`
+while still on the feature or bug branch the changes belong to. When the changes on your feature
+or bug branch are committed, you'll notice that when you are on master or another branch with a
+different timeline, the commit with those changes will not be present. The master branch only has
+the code from the most recent commit relative to the master timeline or branch. The code from our
+`isolating-new-feature` is tucked away in that branch, waiting patiently in isolation from the
+rest of your code in `master` until the feature is considered complete.
 
-```
-From github.com:aviflombaum/mission-critical-application
-   bfe50fc..0ae1da2  master     -> origin/master
- * [new branch]      remote-feature-branch -> origin/remote-feature-branch
-```
+The final step of completing the `isolating-new-feature` work sprint is to merging that timeline
+into the master timeline.
 
-The first line, `From github.com:aviflombaum/mission-critical-application` is informing us which remote our `git fetch` updated from, namely, the remote repository located at: https://github.com/aviflombaum/mission-critical-application
+## Explain Merging branches
 
-When we `fetch` with git, we are asking to copy all changes on the remote to our local git repository, but not actually integrate any. The next line, `bfe50fc..0ae1da2  master     -> origin/master` is telling us that a new commit was found in `origin/master`. `origin/master` means the GitHub version of `master`. Even though git fetched a new commit from `origin/master`, it did not merge it into the local master.
+Now that you have some additions to the code that you'd like to combine back with the master set of
+code, the goal is to bring the timeline of commits that occurred on the `isolating-new-feature`
+branch into the `master`. By merging the timelines, `master` will have all of the commits from the
+`isolating-new-feature` branch as though those events occurred on the `master` timeline.
 
-![Fetch without integration](https://dl.dropboxusercontent.com/s/iy2jovft8ykrxbd/2015-11-02%20at%202.08%20PM.png)
+When merging a branch with `git merge`, it's important to be currently working on your target branch,
+the branch you want to move into. The first step for our `isolating-new-feature` merge is to checkout
+`master` because that is where you want the commits to end up.
 
-Our remote copy on GitHub has a file, `remote-bug-fix`, presumably some code that another developer pushed up to our remote version of the `master` branch to fix a bug. Even after we fetched, our local copy still doesn't appear to have that file.
+When performing `git merge`, `git` will ask you to create a commit to reflect that you've done
```

Comment:
Let's not offer them this path. Direct them to `git merge -m "merge in feature add rainbow-background`"

so basically cut the hypothetical.